Fix for running on windows

jokerslab opened the pull request:
Changed path separator in util/remove.js
Added Dev-dependency on cross-env to allow npm run scripts to set
Environment variables.

schovi commented on dev-env/manifest/processor/assets.js:
```diff
@@ -30,8 +30,10 @@ const processAsset = function(object, key, buildPath) {
-  object[key] = buildAssetPath
+  console.log(buildAssetPath);
+  console.log(buildAssetsDir);
+  console.log(assetPath);
```
Can you remove these logs?

schovi commented on package.json:
```diff
@@ -5,8 +5,8 @@
-    "dev": "NODE_ENV=development ./node_modules/.bin/babel-node ./dev-env/dev.js",
-    "build": "NODE_ENV=production ./node_modules/.bin/babel-node ./dev-env/build.js"
+    "dev": "cross-env NODE_ENV=development babel-node ./dev-env/dev.js",
+    "build": "cross-env NODE_ENV=production babel-node ./dev-env/build.js"
```
I have personal problem with using babel-node or cross-env globally. Users easily forget that, or install different versio. There should be some way how to run bin from `./node_modules/.bin/...`

schovi replied:
Maybe this can be solution
`$(npm bin)/<binary-name> [args]`
example: `$(npm bin)/babel-node install` (except that slash which is incompatible with windows :) )

jokerslab replied:
npm scripts automatically add the path to the dependencies/dev dependencies when run... no need to install the node module globally. 

https://docs.npmjs.com/misc/scripts#path

> If you depend on modules that define executable scripts, like test suites, then those executables will be added to the PATH for executing the scripts. So, if your package.json has this:
> 
> ```
> { "name" : "foo"
> , "dependencies" : { "bar" : "0.1.x" }
> , "scripts": { "start" : "bar ./test" } }
> ```
> 
> then you could run npm start to execute the bar script, which is exported into the node_modules/.bin directory on npm install.

schovi commented:
What about using `dotenv` https://www.npmjs.com/package/dotenv
You can keep your env variables in one file just for you.

Commit:
Additional fixes for running on windows

Added Dev-dependency on cross-env to allow npm run scripts to set
Environment variables and fix image paths

colthreepv commented:
This PR works wonders by the way

schovi commented:
Sorry, I totaly forgot on this PR. I tested it and it works ofcourse. Thanks @jokerslab for teaching me, that npm looks into `node_modules/.bin/` first.
